### DISCUSSION TITLE
ERROR RESTY Get "https://api.coingecko.com/api/v3/coins/list": net/http: fetch() failed: Failed to execute 'fetch' on 'Window': Failed to read the 'signal' property from 'RequestInit': Failed to convert value to 'AbortSignal'

### DISCUSSION THREAD

**Discussion by oliverlj** [go-resty/resty#540] [General]
Hello,

We use resty which is been compiled to wasm and executed within an ember application.

In test, if we setup mock fetch like pollysjs, we see a lot of error like this one : 
ERROR RESTY Get "https://api.coingecko.com/api/v3/coins/list": net/http: fetch() failed: Failed to execute 'fetch' on 'Window': Failed to read the 'signal' property from 'RequestInit': Failed to convert value to 'AbortSignal'., Attempt 1

I don't know if this is Resty related and what to do ?

**Comment by moorereason** [comment]
Can you reproduce the errors with just `net/http`? If not, can you provide a small git repository that reproduces the error?

**Reply by oliverlj** [reply]
I have investigated a little more and i have a package "ember-fetch" in my project which maybe changed the global fetch.

without it, no more issue. Other way to do will be in net/http to check if signal is an instance of AbortSignal
